README.md

```diff
@@ -1,2 +1,60 @@
 # DashJS
-Dash library for JavaScript/TypeScript ecosystem (Wallet, DAPI, Primitives, BLS, ...)
+
+[![Package Version](https://img.shields.io/github/package-json/v/dashevo/dashjs.svg?&style=flat-square)](https://www.npmjs.org/package/@dashevo/dashjs)
+[![Build Status](https://img.shields.io/travis/com/dashevo/dashjs.svg?branch=master&style=flat-square)](https://travis-ci.com/dashevo/dashjs)
+
+> Dash library for JavaScript/TypeScript ecosystem (Wallet, DAPI, Primitives, BLS, ...)
+
+
+## Table of Contents
+
+- [State](#state)
+- [Principles](#principles)
+- [Install](#install)
+- [Usage](#usage)
+- [License](#license)
+
+## State
+
+Under active development.
+
+## Principles
+
+Dash is a powerful new peer-to-peer platform for the next generation of financial technology. The decentralized nature of the Dash network allows for highly resilient Dash infrastructure, and the developer community needs reliable, open-source tools to implement Dash apps and services.
+
+## Install
+
+```sh
+npm install @dashevo/dashjs
```

Comment:
I thought we especially secured https://www.npmjs.com/package/dash and were going to use it for this project?

Reply:
Using @dashevo has a benefit on the configuration of it (it's under a npm org account). 
What do you suggest ? Switch on /dash ? Mirroring seems not a possible options :x

Reply:
@dalibran @shumkov @antouhou as I remember the plan was all along to be able to just use
`npm install dash` for the SDK. Have there been any changes to this?

Reply:
Well, the whole repo is called DashJS, so even without the @dashevo org, I was expecting at least this. 
And then there is the whole maintain process having it elsewhere. 
The thing we really did not want was a 4yr old fork of bitpay or whatever it was under the name dash as one doing npm i dash might end up downloading something that is not working. 
So may initial guess would to put version there with a deprecate message leading to a @dashevo owned npm ?

Reply:
The plan is to still use `npm install dash` and refer to all of the client libraries as the Dash SDK. I'd prefer to not distinguish between the other implementations by different names (i.e. DashJS or DashJ or go-dash), since they should all of the same interface and therefore same developer experience. 

We've discussed naming them using the same convention in Github so that they appear alphabetically for easier searching -- something like `dash-js`, `dash-go`, `dash-py`

@nmarley can provide more context as to the best repo naming convention

Reply:
We discussed and started using language prefixes because there can be multiple implementations of a library or standard, e.g. `js-dpp`, `ios-dpp`, and `android-dpp` for example. This prefix convention was copied from the IPFS org (cf `go-ipfs` and `js-ipfs`, etc.) and as a bonus makes it easier to filter language-specific repos in a search.